### DISCUSSION TITLE
Can we make the MediaDC to be an Similarity Image Search

### DISCUSSION THREAD

**Discussion by weasam** [cloud-py-api/mediadc#70] [Ideas]
I know my  Idea might be crazy for upload an image to nextcloud to search by image similarity, I don't know how to code, I just learned some the logic of the algorism for image search or recognition. to upload an image to search in nextcloud file make take very much of Server resource, but it may be a good reason to try. 
 
As digital assets management, my use case would be like this:
 1. example I have a mud with wooden texture, I want to search whether my nextcloud have or not, I upload it and search.

**Comment by andrey18106** [comment]
@weasam Hi!

Sorry for such a late answer. Really nice idea, similar to [this issue](https://github.com/cloud-py-api/MediaDC/issues/68), or even as a complement of it. Therefore further discussion about this theme will be here as soon as we'll have recourses for that to implement.

**Reply by weasam** [reply]
Thanks for your reply. I guess will take a lot server load, it may need Elasticsearch to get this feature
